src/gwt/src/org/rstudio/studio/client/panmirror/dialogs/PanmirrorEditLinkDialog.ui.xml

```diff
@@ -0,0 +1,17 @@
+<!DOCTYPE ui:UiBinder SYSTEM "http://dl.google.com/gwt/DTD/xhtml.ent">
```

Comment:
Is this used? (looks like parent class never binds it)

Reply:
Removed here: https://github.com/rstudio/rstudio/commit/6b9ef6ea3bfbd04925796c63af1daa3e1c7d2d9d